docs/howto/build-run.md

```diff
@@ -76,6 +76,20 @@ next person with a setup like yours.
 
 ## Troubleshooting
 
+### Bundling Failure: Unable to resolve module...
+
+When trying to bundle the application via Metro after pulling new code, you may
```

Comment:
It is good that you give a more concrete example and solution, but that removes the more general description that encompasses more use-cases, as this `reinstall` is not only useful for the 'new dependency added' case.

Reply:
I think 'new or updated dependencies' would be more encompassing and explicit; I've changed it to say that.

In what other cases other than new or updated dependencies might `reinstall` be used?

I agree that mentioning `reinstall` at the end as a possible general catch-all would be a helpful tip for troubleshooting; I could put that mention at the end back in, and reword the commit and this PR to be specific to the case of pulling new or updated dependencies.